etc/p3-config/p3-config.yml

```diff
@@ -197,3 +197,88 @@ p3_flavor_compute_A_magnum:
   disk: 271
   vcpus: 64
   ephemeral: 100
+
+###############################################################################
+# Software images for P-cubed
+
+p3_images:
+  - "{{ p3_image_centos }}"
+  - "{{ p3_image_ubuntu }}"
+  - "{{ p3_image_fedora }}"
+  - "{{ p3_image_openhpc }}"
+  - "{{ p3_image_mlnx_ofed }}"
+
+# Latest CentOS image
+p3_image_centos:
+  name: "CentOS7"
+  elements:
+    - "centos7"
+    - "epel"
+    - "selinux-permissive"
+    #- "alaska-extras"
+    - "dhcp-all-interfaces"
+  env:
+    DIB_ALASKA_DELETE_REPO: "y"
+    DIB_ALASKA_PKGLIST: "pam-python pam-keystone"
+
+# Latest Ubuntu image
+p3_image_ubuntu:
+  name: "Ubuntu16"
+  elements:
+    - "ubuntu"
+
+# Latest Fedora Core image
+p3_image_fedora:
+  name: "FedoraCore"
+  elements:
+    - "fedora"
+    - "selinux-permissive"
+    #- "alaska-extras"
+  env:
+    DIB_ALASKA_DELETE_REPO: "y"
+    DIB_ALASKA_PKGLIST: "pam-python pam-keystone"
+
+# Build of OpenHPC on CentOS
+p3_image_openhpc:
+  name: "CentOS7-OpenHPC"
+  elements:
+    - "centos7"
+    - "epel"
+    - "openhpc"
+    - "selinux-permissive"
+    #- "alaska-extras"
+    - "dhcp-all-interfaces"
+  env:
+    DIB_OPENHPC_GRPLIST: "ohpc-base-compute ohpc-slurm-client 'InfiniBand Support'"
+    DIB_OPENHPC_PKGLIST: "lmod-ohpc mrsh-ohpc lustre-client-ohpc ntp"
+    DIB_OPENHPC_DELETE_REPO: "n"
+    DIB_ALASKA_DELETE_REPO: "y"
+    DIB_ALASKA_PKGLIST: "pam-python pam-keystone"
+
+# Build of Mellanox OFED on CentOS
+p3_image_mlnx_ofed:
+  name: "CentOS7-OFED4"
+  elements:
+    - "centos7"
+    - "epel"
+    - "mlnx-ofed"
+    - "selinux-permissive"
+    #- "alaska-extras"
+    - "dhcp-all-interfaces"
+  env:
+    DIB_MLNX_OFED_VERSION: "4.0-2"
+    DIB_MLNX_OFED_REPO: "http://127.0.0.1/repo/MLNX_OFED_LINUX-4.0-2.0.0.1-rhel7.3-x86_64"
+    DIB_MLNX_OFED_DELETE_REPO: "y"
+    DIB_ALASKA_DELETE_REPO: "y"
+    DIB_ALASKA_PKGLIST: "pam-python pam-keystone"
+
+# This creates a git checkout in the local user's home directory
+p3_image_stackhpc_elements: "{{ ansible_env.PWD }}/stackhpc-image-elements"
+
+p3_image_git_elements:
+  - repo: "https://github.com/stackhpc/stackhpc-image-elements.git"
+    local: "{{ p3_image_stackhpc_elements }}"
+
+# Path to include
+p3_image_elements: "{{ p3_image_stackhpc_elements }}/elements"
```

Comment:
Another minor point, it's a shame that the user needs to generate the elements path manually when we have all the information in os_image_git_elements except for the relative path to the elements directory.